README.md

```diff
@@ -1,832 +1,390 @@
+	https://img.shields.io/static/v1?style=flat&logo=linkedin&logoColor=0077B5&label=LinkedIn&labelColor=FFFFFF&color=0077B5&message=Vikrant%20Bhat
+)](https://linkedin.com/in/vikrant-bhat-2b6221189)
+<br>
+[![Vikrant Bhat personal website](
+	https://img.shields.io/static/v1?style=flat&label=Website&labelColor=FFFFFF&color=333333&message=underreacted.netlify.app
+)](https://underreacted.netlify.app)
+
+### Damian Szczypka
+[![Damian Szczypka GitHub profile](
+	https://img.shields.io/github/followers/sthiepaan?style=flat&logo=github&logoColor=181717&label=GitHub&labelColor=FFFFFF&color=181717
+)](https://github.com/sthiepaan)
+[![Damian Szczypka Twitter profile](
+	https://img.shields.io/twitter/follow/sthiepaan?style=flat&logo=twitter&logoColor=1DA1F2&label=Twitter&labelColor=FFFFFF&color=1DA1F2
+)](https://twitter.com/sthiepaan)
+[![Damian Szczypka LinkedIn profile](
+	https://img.shields.io/static/v1?style=flat&logo=linkedin&logoColor=0077B5&label=LinkedIn&labelColor=FFFFFF&color=0077B5&message=Damian%20Szczypka
+)](https://linkedin.com/in/damianszczypka)
+<br>
+[![Damian Szczypka personal website](
+	https://img.shields.io/static/v1?style=flat&label=Website&labelColor=FFFFFF&color=333333&message=damianszczypka.com
+)](http://damianszczypka.com)
+
+If you'd like to see everyone who contributed to this project,
+view the [contributions page][URL_GH_CONTRIBUTORS]!
+<br>
+Thank you to everyone who contributes! 🙌
+
+[↑ Back to menu][URL_TOC]
 
 ---
 
 ## License
 
-![GitHub license](https://img.shields.io/github/license/bhatvikrant/world-countries-capitals.svg?style=for-the-badge&logo=github)
+Copyright &copy; 2020 [Vikrant Bhat](https://github.com/bhatvikrant).
+<br>
+This project is [MIT][LICENSE] licensed.
 
-This project is licensed under the MIT License, Copyright (c) 2020 Vikrant Bhat.
+[↑ Back to menu][URL_TOC]
 
 ---
 
-## Let's Connect
-
-[![Twitter Follow](https://img.shields.io/twitter/follow/vikrantbhat1022.svg?style=for-the-badge&logo=twitter)](https://twitter.com/vikrantbhat1022) [![GitHub followers](https://img.shields.io/github/followers/bhatvikrant.svg?label=Follow&style=for-the-badge&logo=github)](https://github.com/bhatvikrant/) [![LinkedIn](https://img.shields.io/static/v1.svg?label=connect&message=@vikrantbhat&color=success&logo=linkedin&style=for-the-badge&logoColor=white&colorA=blue)](https://www.linkedin.com/in/vikrant-bhat-2b6221189/)
-
-<hr>
-
-[![forthebadge](https://forthebadge.com/images/badges/built-with-love.svg)](https://github.com/bhatvikrant/world-countries-capitals) [![forthebadge](https://forthebadge.com/images/badges/made-with-javascript.svg)](https://github.com/bhatvikrant/world-countries-capitals) [![forthebadge](https://forthebadge.com/images/badges/makes-people-smile.svg)](https://github.com/bhatvikrant/world-countries-capitals) [![forthebadge](https://forthebadge.com/images/badges/check-it-out.svg)](https://github.com/bhatvikrant/world-countries-capitals)
+<!-- TODO: Prepare .github template files -->
```

Comment:
I think this TODO is done, we can remove the comment or am I missing something?

Reply:
> I think this TODO is done [...]

My bad, forgot to remove that one. Thanks!

> [...] we can remove the comment or am I missing something?

I don't think we need more templates. At least for now 😉